docs/recipes/recipes.ipynb

```diff
@@ -103,6 +103,276 @@
     "Using `bind_and_call` guarantees that the file gets written and that it gets written after the pipeline.\n",
     "The latter prevents providers from accidentally relying on the file."
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "60f9b2a8-0557-43da-a8d2-41df2794bd49",
+   "metadata": {},
+   "source": [
+    "## Continue from intermediate results\n",
+    "\n",
+    "It is a common need to be able to continue the pipeline from some intermediate result computed earlier.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "aa1df625-9f6a-44d3-9169-1c79ff8e647f",
+   "metadata": {
+    "jp-MarkdownHeadingCollapsed": true
+   },
+   "source": [
+    "### Setup\n",
```

Comment:
Before the full and working example, should we give a TL;DR?

```python
data = pipeline.compute(CleanData)
pipeline[CleanData] = data
result = pipeline.compute(Result)
```

Reply:
Yeah that makes sense